src/models/activityReportCollaborator.js

```diff
@@ -27,6 +28,11 @@ export default (sequelize, DataTypes) => {
     fullName: {
       type: DataTypes.VIRTUAL,
       get() {
+        if (!this.user) {
+          const { stack } = new Error();
+          auditLogger.error(`Access attempt to undefined user for userID: ${this.userId}, stack: ${stack}`);
+          return '';
+        }
```

Comment:
This is unrelated but still a good fix, correct?

Reply:
This is somewhat related, as we are trying to gather more information about what is causing TTAHUB-3136, which crashes the server. If this particular spot is where the error comes from, it would be helpful to determine why the user object is undefined.